Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM aetrnty/builder as builder
 
+# Add required files to download and compile only the dependencies
+ADD rebar.config rebar.lock Makefile rebar3 VERSION /app/
```

Comment:
I see rebar.config.script not included meaning "release" not set correctly. I expect no impact for building deps now.
Though, in order to be more robust to future changes in rebar.config.script, may I suggest including it here so, if it gets to do more and impact the deps, this Dockerfile is not affected.

Reply:
I agree that this script might impact the dependancies in future.

@amir-hadi could you please include it?

Reply:
@dincho Added the requested script in the first step.